ndio/remote/testND.py

```diff
@@ -0,0 +1,5 @@
+import ndio.remote.neurodata as ND
```

Comment:
We use py.test for all of our tests, so they need to adhere to that convention. The other tests in the folder have some great examples and docs can be found [here](http://doc.pytest.org/en/latest/contents.html).